frontend/public/components/app.jsx

```diff
@@ -159,6 +159,11 @@ class App_ extends React.PureComponent {
                 onPerspectiveSelected={this._onNavSelect}
               />
             }
+            skipToContent={
+              <SkipToContent href={`${this.props.location.pathname}#content`}>
```

Comment:
This will drop any query parameters in the URL I believe.

Reply:
Very good point, I hadn't considered that. I just updated it to add that consideration!